src/components/NewForm/FormSettings.js

```diff
@@ -0,0 +1,94 @@
+import { Form, Input, Switch } from "antd";
+import { useEffect } from "react";
+
+const FormSettings = (props) => {
+  const { form, onFormFinish, onFinishFailed } = props;
+
+  useEffect(() => {
+    if (form.getFieldValue("selfSign") === undefined) {
+      form.setFieldValue("selfSign", false);
+    }
+    if (form.getFieldValue("showOnGlobal") === undefined) {
+      form.setFieldValue("showOnGlobal", true);
+    }
+    console.log("value", form.getFieldValue("showOnGlobal"));
+  });
+
+  function handleNameChange(event) {
```

Comment:
write a single function, pass fieldType string

Reply:
thanks, will take it up in future PRs